src/coreclr/jit/targetamd64.h

```diff
@@ -167,11 +167,18 @@
   #define REG_FLT_CALLEE_SAVED_FIRST   REG_XMM6
   #define REG_FLT_CALLEE_SAVED_LAST    REG_XMM15
 
+  #define RBM_LOWINT              RBM_ALLINT_INIT
+  #define RBM_HIGHINT             (RBM_R16|RBM_R17|RBM_R18|RBM_R19|RBM_R20|RBM_R21|RBM_R22|RBM_R23)
+
+  #define RBM_ALLINT_INIT         (RBM_INT_CALLEE_SAVED | RBM_INT_CALLEE_TRASH_INIT)
+  #define RBM_ALLINT              get_RBM_ALLINT()
+  #define RBM_INT_CALLEE_TRASH_STATIC_ALL (RBM_INT_CALLEE_TRASH_INIT | RBM_HIGHINT)
```

Comment:
`RBM_INT_CALLEE_TRASH_INIT ` serves similar purpose as other `#define` with name `*_STATIC_*` in it. Can you have consistent nomenclature, either use `*_INIT_*` or `*_STATIC_*` in their names?

Reply:
that makes sense and `*_INIT` seems fine to me....I am little uncomfortable with the work `_STATIC_` in the other `#define`, so if you can think of a better name that will be great.

Reply:
Updated names by removing `STATIC`. `RBM_INT_CALLEE_ALL`, `RBM_ALLINT_ALL` describes these I think